Units/parser-javascript.r/js-es6-class.d/input.js

```diff
@@ -1,18 +1,18 @@
 
 class Class1
 {
-  method1(arg1,arg2)
-  {
-  }
+	method1(arg1,arg2)
+	{
+	}
   
-  method2(arg1,arg2)
-  {
-  }
+	method2(arg1,arg2)
+	{
+	}
 }
 
 class Class2
 {
-  method3(arg = 10)
-  {
-  }
+	static method3(arg = 10)
+	{
+	}
```

Comment:
I think I'll re-make a cleaner pull request with these changes in the next days.